src/sentry/static/sentry/app/views/performance/vitalDetail/utils.tsx

```diff
@@ -33,20 +33,18 @@ export enum VitalState {
   GOOD = 'Good',
 }
 
-export const vitalStateColors = {
-  [VitalState.POOR]: theme.red300,
-  [VitalState.MEH]: theme.yellow300,
-  [VitalState.GOOD]: theme.green300,
+export const vitalStateColors: Record<VitalState, Color> = {
+  [VitalState.POOR]: 'red300',
+  [VitalState.MEH]: 'yellow300',
+  [VitalState.GOOD]: 'green300',
 };
 
-function IconCircledCheckmark(props) {
-  return <IconCheckmark {...props} isCircled />;
-}
-
-export const vitalStateIcons = {
-  [VitalState.POOR]: IconFire,
-  [VitalState.MEH]: IconWarning,
-  [VitalState.GOOD]: IconCircledCheckmark,
+export const vitalStateIcons: Record<VitalState, React.ReactNode> = {
+  [VitalState.POOR]: <IconFire color={vitalStateColors[VitalState.GOOD]} />,
+  [VitalState.MEH]: <IconWarning color={vitalStateColors[VitalState.GOOD]} />,
+  [VitalState.GOOD]: (
+    <IconCheckmark color={vitalStateColors[VitalState.GOOD]} isCircled />
```

Comment:
Should these all be the same color?

Reply:
absolutely not, but i fixed that already

Reply:
could you review it, uh... again again? :/